tests/test_ops_qubit.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the available built-in discrete-variable quantum operations.
+"""
+from functools import reduce
+
+import pytest
+import numpy as np
+
+from pennylane.ops import qubit
+from pennylane.plugins.default_qubit import Rot3
+
+
+class TestHadamard:
+    """Test functions for Hadamard class."""
+    def test_hadamard_matrix(self):
```

Comment:
We also have a tolerance fixture available in `conftest.py` :) So it might be good to add it as a test function argument, and then use it in the `np.allclose` below.

Reply:
Added `atol=tol` and `rtol=0` to `np.allclose`.